lib/l10n/app_en.arb

```diff
@@ -915,6 +921,13 @@
     "s_nfc_dialog_fido_rename_fingerprint": "Action: rename fingerprint",
     "s_nfc_dialog_fido_failure": "FIDO operation failed",
 
+    "@_nfc": {},
+    "s_nfc_ready_to_scan": "Ready to scan",
+    "s_nfc_scanning": "Scanning\u2026",
+    "s_nfc_tap_your_yubikey": "Tap your YubiKey",
+    "l_nfc_failed_to_scan": "Failed to scan, try again",
+    "s_nfc_done": "Done",
```

Comment:
This could be just `s_done` and could then be reused in other places.

Reply:
fixing!